README.md

```diff
@@ -155,7 +155,21 @@ $ make clean
 
 ### Building with Doxygen
 
-If you want to build the Pico C SDK Doxygen documentation alongside the main documentation site you can do so with,
+If you want to build the Pico C SDK Doxygen documentation alongside the main documentation site, first install the following dependencies:
+
+NOTE: For the time being, building with any Doxygen version other than 1.8.17 will result in formatting errors in the built site.
+
+```
+$ brew install doxygen@1.8.17 graphviz
```

Comment:
`doxygen@1.8.17` doesn't seem to be a formula or cask that exists?

```
$ brew info doxygen@1.8.17
Error: No available formula with the name "doxygen@1.8.17". Did you mean doxygen?
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

We could instead provide a [Homebrew tap](https://docs.brew.sh/Taps) that provides this formula based on  the [old Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/e0b3d409123fd11e5d914a1c281bae387a8aecd9/Formula/doxygen.rb)?